tests/acceptance/task-comments-test.js

```diff
@@ -15,14 +25,15 @@ test('Task comments are displayed correctly', function(assert) {
 
   server.createList('comment', 4, { taskId: task.id });
 
-  taskPage.visit({
+  renderPage();
+  page.visit({
     organization: organization.slug,
     project: project.slug,
     number: task.number
   });
 
   andThen(() => {
-    assert.equal(taskPage.comments().count, 4, 'The correct number of task comments is rendered');
```

Comment:
I'm getting an error that says "died on test#1" then shows `Error: Assertion Failed: ":first" matched more than one element. If this is not an error use { multiple: true }` 

This is my first assert and I'm not seeing `first:` in the code. Only thing I can think of is that somehow it's trying to return the first comment in the list but is catching something else with it? In the past, when I've seen this error, it's because I was re-rendering the objects through a `export default create` on the page object. So, I'm assuming that I've got code somewhere that's returning multiple objects, but I'm not sure how and what because I can't even track down where `:first` is defined.

Reply:
To explain why you're not seeing `:first` in the code.

If you look at the docs for `collection()`, there are some examples there:

http://ember-cli-page-object.js.org/docs/v1.12.x/api/collection

If you define a collection as 

```javascript

items: collection({
  scope: 'foo',
  itemScope: 'bar'
})
```

What this means is, you're saying

- i have a bunch of elements of the same type on my page
- they are all contained within an element with the class of 'foo'
- each individual one of those items is carrying the class 'bar'

Since you said that, 
- `page.items().count` internally calls `$('.foo .bar').length`
- `page.items(1)` internally calls `$('.foo .bar:eq(1)')`

Now, all that's fine now that you fixed the page object, but your old one had some weird error, which I can't figure out myself, which caused the `ember-cli-page-object` internals to fallback on their default selector - `:first`, which then also somehow returned multiple objects, which then, by default, unless you manually set it up otherwise, will error out.

So you might not have seen `:first` yourself, but it was in there, same as how 
'.foo .bar:eq(1)' is in there, even though you're not the one writing it.